spec/controllers/static_pages_controller_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe StaticPagesController, type: :controller do
```

Comment:
I wouldn't bother with controller specs, I'd replace these with feature specs.  Ensure you have them for static pages and also articles